css/index.css

```diff
@@ -1,3 +1,451 @@
+  border-bottom: 1px solid #c9c9c9;
+}
+
+.service-desc {
+  max-width: 48%;
+  background-color: #F4F3F4;
+  border: 1px solid #c9c9c9;
+  padding: 20px;
+  margin: 20px 0;
+}
+
+.service-desc h2 {
+font-size: 20px;
+}
+
+button {
+  display: block;
+  margin-top: 15px;
+  padding: 8px 15px;
+  background-color: #fff;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.services-bottom-row {
+	display: flex;
+	justify-content: space-between;
+	align-items: flex-start;
+	margin-top: 40px;
+}
+
+.services-bottom-row p {
+	padding-bottom: 20px;
+}
+
+/* Start of code */
+
+a {
+    
+    text-decoration: none;
+    color: gray;
+   
+}
+
+hr {
+    margin-bottom: 60px;
+    border: 1px black solid;
+    
+}
+
+.bottom {
+    margin-top: 60px;
+   
+}
+
+.logo {
+    display: flex;
+    flex-wrap: wrap;
+    width: 130px;
+    align-content: center;    
+    
+}
+
+
+.cta-text {
+    display:inline-block;
+    width: 300px;
+    text-align: center;
+    margin: 70px   ;
+    position:relative;
+    bottom: 60px;
+    font-size: 100px;
+
+}
+
+.circle {
+    display: inline-block;
+    margin-top: 150px
+}
+
+button {
+    margin-left: 100px 
+}
+
+
+.main-content{ 
+    padding-top: 30px;
+    display:inline-block;
+    width: 97.6%;
+    
+}
+
+ .bottom-content{ 
+    
+    display:inline-block;
+    width: 100%;
+    
+ }
+.text-content{
+    display: flex;
+    flex-wrap: wrap;
+    display:inline-block;
+    width: 48.3%;
+    margin-right: 12px;   
+}
+
+.text-content2{
+    display:inline-block;
+    width: 32.9%;
+   
+}
+
+footer {
+    
+    margin-left: 38%;
+} 
+
+hr .media {
+    display: none
+}
+
+
+
+Tablet Media View
+
+    @media (max-width: 900px) {
+
+   
+        .container {
+            display: flex;
+            flex-wrap: wrap;
+            width:100%;
+            justify-content:center;
+            background-color: yellow;
+    
+            
+        }
+
+        nav {
+            display: flex;
+            flex-wrap: wrap;
+            align-items: flex-start;
+            width: 100%;
+            border: red solid 1px;
+            height: 250px
+        }
+
+
+        .text-content {
+            display: inline-block;
+            margin-left: 20px;
+            width: 46%;
+        }
+    
+        .logo {
+            display: flex;
+            flex-wrap: wrap;
+            align-items:center;
+            width: 80%;
+          
+        }
+       
+        hr {
+            display: none;
+        }
+
+
+    a {
+        display: flex;
+        flex-wrap: wrap;
+        margin: 12px
+    }
+
+    hr { 
+        width: 100%
+        
+    }
+    .circle {
+        display:none
+    }
+
+   
+    .contact {
+        
+        float:right;
+        margin-bottom: 30px;
+        padding-top: 90px;
+        margin-right: 650px
+    }
+ 
+    footer {
+
+        padding-bottom: 900px;
+        padding-right:300px;
+        
+        }
+
+}
+
+/* Mobile Media View */
+
+@media (max-width: 766px) {
```

Comment:
I really like how you added your own style to the mobile view. It looks good and you easily got your navigation items into a stacked layout. Your mobile view was executed well and looks sharp.